lib/clients/helm.js

```diff
@@ -38,6 +40,9 @@ class Helm {
     this.botRoomId = process.env.MATRIXBOT_ROOM_ID || '';
     this.botHomeserver = process.env.MATRIXBOT_HOMESERVER || '';
     this.cloudflareEmail = process.env.CLOUDFLARE_EMAIL || '';
+    this.opsgenieEnabled = process.env.OPSGENIE_ENABLED || false;
```

Comment:
I think it would be better if instead we should get a config option like for monitoring, like `opsgenie: true`, then use `this.config.opsgenie`.